index.html

```diff
@@ -0,0 +1,145 @@
+    </ul>
+  </nav>
+</header>
+<ul class="navigation__mobile">
+  <li><a href=""><img id="close-menu" class="close-menu" src="./images/close.png" alt="close"></a></li>
+   <li><a href="./about.html" class="navigation__mobile__links externalLink">About</a></li>
+   <li><a onclick="openLinks('#program');" class="navigation__mobile__links internalLinks">Program</a></li>
+   <li><a onclick="openLinks('#speakers');" class="navigation__mobile__links internalLinks">Speakers</a></li>
+</ul>
+<section class="home p-4">
+  <p class="home__para1">
+    "Greetings, students of Philosophy"
+  </p>
+  <h1>
+    Masters of Philosophy™ <br>
+    ANNUAL CONGRESS OF DISCOURSE <br>
+    2022 <br>
+  </h1>
+  <p class="home__para2 p-3">
+    An event for all who enjoy the timeless lectures 
+    and scholarly wisdom of Ancient Philosophers brought to life 
+    through the brilliance of hologram technology. 
+  </p>
+  <h2>
+    2020.06.19 (MON) ~ 20(TUES)
+  </h2>
+  <h3>
+    @ The Seneca Hall of Congress, Athens
+  </h3>
+</section>
+
+<section id="program" class="program p-4">
+  <h2 class="text-center p-3 ml-auto mr-auto">
+    Main Program
+  </h2>
+  <div class="program__container m-3">
+    <article class="program__container__content m-2 align-items-center">
+      <img src="./images/hologram.png" alt="hologram">
+      <h3>
+        Lecture
+      </h3>
+      <p>
+        Watch and listen to the teachings of the greatest Philosophers to ever grace the earth
+        using world-class hologram technology.
+      </p>
+    </article>
+    <article class="program__container__content m-2 align-items-center">
+      <img src="./images/itinerary.png" alt="itinerary">
+      <h3>
+        Tour
+      </h3>
+      <p>
+        Take a tour of the refurbished Seneca Hall of Congress and marvel at
+        its state of the art architectural design.
+      </p>
+    </article>
+    <article class="program__container__content m-2 align-items-center">
+      <img src="./images/forum.png" alt="forum">
+      <h3>
+        Forum
+      </h3>
+      <p>
+        Engage in witful discourse with fellow philosophy scholars.
+      </p>
+    </article>
+    <article class="program__container__content m-2 align-items-center">
+      <img src="./images/exhibition.png" alt="exhibition">
+      <h3>
+        Exhibition
+      </h3>
+      <p>
+        Take to admire the works of both ancient and modern Philosophers in 
+        an artistic Exhibition of talents.
+      </p>
+    </article>
+    <article class="program__container__content m-2 align-items-center">
+      <img src="./images/shop.png" alt="shop">
+      <h3>
+        Shop
+      </h3>
+      <p>
+        Purchase some immaculately crafted souvenirs and memorabilia and 
+        take a piece of Athens home with you.
+      </p>
+    </article>
+    <button>
+      Book your Ticket Now
+    </button>
+  </div>
+  <a href="">See Full Program</a>
+</section>
+
+<section id="speakers" class="speakers p-2 m-2">
```

Comment:
- [ ] Same as the previous section please reduce the length of the divider so your design is similar to the CC design
- [ ] Please stick to the provided design, you can add the little squares behind the images in this section and remove the background gray color on the philosopher's details 

|Your design|CC design|
|--|--|
|![Screenshot_20220526_204159](https://user-images.githubusercontent.com/25227022/170565093-fb70e83f-6dfc-42fc-a353-4b8fefe9451d.png)|![Screenshot_20220526_204658](https://user-images.githubusercontent.com/25227022/170565626-e574199d-9883-4015-be30-c2f135abf7cc.png)|